src/Symfony/Bundle/FrameworkBundle/Resources/config/web.xml

```diff
@@ -51,6 +51,11 @@
             <tag name="controller.argument_value_resolver" priority="-150" />
         </service>
 
+        <service id="argument_resolver.controller_not_registered_as_service" class="Symfony\Component\HttpKernel\Controller\ArgumentResolver\ControllerNotRegisteredAsServiceValueResolver">
```

Comment:
maybe NotTaggedControllerValueResolver?

Reply:
Renamed class and its test to the new name.